migrations/versions/fea4af6bf9ce_renaming_datadate_to_date.py

```diff
@@ -0,0 +1,30 @@
+"""Renaming dataDate to date
+
+Revision ID: fea4af6bf9ce
+Revises: 01f76bf60f51
+Create Date: 2020-05-27 22:51:45.711845
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'fea4af6bf9ce'
+down_revision = '01f76bf60f51'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('coreData', sa.Column('date', sa.Date(), nullable=False))
+    op.drop_column('coreData', 'dataDate')
```

Comment:
Unfortunately, this fails if the database contains any data, because it tries to add a non-nullable column containing null to all the existing rows: 

> [SQL: ALTER TABLE "coreData" ADD COLUMN date DATE NOT NULL]
> (Background on this error at: http://sqlalche.me/e/gkpj)

(it will not actually provide any background on the error if you go to that URL)

This should rename the existing column instead, preserving the data inside (this is, ironically enough, the opposite process we did to rename `date` to `dataDate` last week 🤦‍♂️
```suggestion
    op.alter_column('coreData', 'dataDate', nullable=False, new_column_name='date')
```

Reply:
whoops - thank you!